compiler/damlc/tests/daml-test-files/Validation.daml

```diff
@@ -16,6 +17,6 @@ main = scenario do
   let v = run do
       head l <?> "fail"
       y <- tail l <?> "fail"
-      pure $ length y
+      return (length y)
```

Comment:
It would be great if you could open an issue for making `$` work and reference it here.

Reply:
The ticket is at https://github.com/digital-asset/daml/issues/4555. I assume by "here" you mean in this PR. IMO, the reference only makes sense in the context of the change but in the context of the resulting code as it looks rather ordinary.

Reply:
Something like “Note that `return $` doesn’t work here, see #4555” is still helpful imho.